src/components/modals/VoteHistoryModal.vue

```diff
@@ -18,21 +18,24 @@
           <td>Nominator</td>
           <td>Nominee</td>
           <td>Type</td>
-          <td>Majority</td>
+          <td>Votes</td>
           <td><font-awesome-icon icon="hand-paper" /> Hand up</td>
+          <td>Passed</td>
```

Comment:
Maybe call it "Majority" instead? Whether or not it passed could be influenced by roles like Thief or Bureaucrat, which could make "Passed" misleading.